app/src/main/java/io/multy/util/Constants.java

```diff
@@ -29,6 +29,11 @@
     public static final String PREF_BACKUP_SEED = "backup_seed";
     public static final String PREF_AUTH = "pref_auth";
     public static final String PREF_EXCHANGE_PRICE = "pref_exchange_price";
+    public static final String PREF_LOCK ="pref_lock";
```

Comment:
Avoid write ununderstandable descriptions. Leave it empty or write complete logical description If constant name isn't enough